reana_workflow_controller/workflow_run_managers/__init__.py

```diff
@@ -21,10 +22,16 @@ class WorkflowRunManager():
         {
             'name': 'SHARED_VOLUME_PATH',
             'value': '/reana'
-        }
+        },
     ]
+
```

Comment:
Please remove the final empty line, so that the variable docstring is attached right after the assignement:

```
x = 1
"""x is this and that"""
```